### ISSUE TITLE
More meta data for data (may be kaggle like tables)

### ISSUE THREAD

**Issue by ycAbout** [CONP-PCNO/conp-portal#186]
## Purpose
To make conp more usable to researchers

## Context
Datasets are large (many G), people cannot get much information before they download the data in the current way. People need to know more about the data beforehand.

## Possible Implementation (optional)
Like a kaggle dataset, list summary information in a table that can directly viewed from the front end.

**Comment by glatard** [comment]
I think a good goal would be to represent as many elements from the DATS model as possible, including access policy (see also #182).

**Comment by glatard** [comment]
Using tags in the dataset page might also be a good graphical way to do it.

**Comment by cmadjar** [comment]
Discussed at the CONP dev call of September 30th, 2020.

@liamocn will investigate kaggle and see if we can use that to display the content of the DATS.json file. Currently only some parts of the DATS.json file is being displayed on the front end but it would be nice to have an `expand` section that shows all the content of the DATS file in a more intuitive and user friendly way that the JSON representation.

**Comment by cmadjar** [comment]
This is blocked by the redesign of the dataset page: #375 and #374
